### PR TITLE
MAID-1010 vault put flow documentation

### PR DESCRIPTION
**Pull request by maqi**
put flow involving Primary, Backup and Sacrificial copies of data

[Review on Reviewable](https://reviewable.io/reviews/maidsafe/maidsafe_vault/78)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @ustulation

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by rossmuir** [comment]
test this please

**Comment by rossmuir** [comment]
test this please

**Comment by maqi** [comment]
close for rebase

File: docs/put_flow.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+--
+<dd>Description: Get D from TempStore or network, then PutRequest(D).</ddt>
+#####DataManager::Replicate
+__DataManager__ ([!TempStoreHas(D) ? NetworkGet(D)])(PutRequest.So(D))
+
+
+
+
+
+
```
Comment (inetic) [inline docs/put_flow.md]: Do we have a legend for this language somewhere online?

**Pull request closed without merging**